container-apache-http-client-bundle/pom.xml

```diff
@@ -68,9 +48,72 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>com.yahoo.vespa</groupId>
-                <artifactId>bundle-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>parse-version</id>
+                        <goals>
+                            <goal>parse-version</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
                 <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Version>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}</Bundle-Version>
+                        <Export-Package>org.apache.hc.*;version=1.0.0;-noimport:=true,org.apache.http.*;version=1.0.0;-noimport:=true</Export-Package>
+                        <Embed-Dependency>*;scope=compile;type=!pom;inline=true</Embed-Dependency>
+                        <Import-Package>!javax.servlet,javax.*,org.slf4j,org.ietf.jgss</Import-Package>
```

Comment:
javax.* seems a bit risky? How future proof is this?

Reply:
It's importing only `javax` packages part of the standard library atm.